src/chi/components/card/card.scss

```diff
@@ -124,7 +124,44 @@
       &.-noBorder {
         box-shadow: none;
       }
+    }
+
+    &.-empty {
+      background-color: transparent;
+      border: dashed rgba(set-color(black), 0.12);
+      transition: all 0.2s;
+
+      & .a-card__content {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin: 1rem;
+
+        & .a-icon {
+          color: set-color(grey, 60);
+          margin-bottom: 1rem;
+        }
 
+        & .-title {
```

Comment:
@davidojedacl @jllr we already have a `a-card__title` class so this will need to change. Let's do what Cyclops does and use a `<p>`. Then remove the font-weight and font-size from the css and keep it generic. So end result will be `<p class="-text -m--0">Add Product</p>`

Reply:
Regarding pseudoclass i will remove, but I put it thinking about test purposes similar to how it is on the buttons or other components
<img width="637" alt="screen shot 2018-12-20 at 10 45 03 am" src="https://user-images.githubusercontent.com/43439742/50277492-de936680-0444-11e9-9cda-08e22a2e50a4.png">